mozilla/activity-stream
# Bug 1516008 - Add pocket layout code behind its own separate pref

Builds on scott's initial patch, but separates stuff out. Not completely done but some good progress. Uses nick's suggested naming convention `DiscoveryStream`.  Based on some of dmose's work in #4624.

### Testing instructions:
1. Ensure the new tab is the regular version by default.
2. In `about:config`, change  `browser.newtabpage.activity-stream.discoverystream.config` to be `{"enabled":true,"layout_endpoint":"https://gist.githubusercontent.com/ScottDowne/164995d9535b4203846048bdee29d169/raw/05804e9e1a9a209b3676509d9752fc8596a7d30c/spoc.json"}`
3. Reset the pref, ensure everything is back to normal.
